modules/planning/tasks/optimizers/piecewise_jerk_speed/piecewise_jerk_speed_nonlinear_optimizer.cc

```diff
@@ -229,5 +231,50 @@ Status PiecewiseJerkSpeedNonlinearOptimizer::Process(
   return Status::OK();
 }
 
+PiecewiseJerkTrajectory1d
+PiecewiseJerkSpeedNonlinearOptimizer::SmoothSpeedLimit(
+    const SpeedLimit& speed_limit) {
+  // using piecewise_jerk_path to fit a curve of speed_ref
+  double delta_s = 2.0;
```

Comment:
This hard coded a fixed `delta_s` of `2.0` and an overall `speed_ref` size of `100`, move them to configs to make it more generic.